docs/contribute.md

```diff
@@ -18,65 +18,38 @@ Prepare your system by installing prerequisites.
 
 ### System libraries
 
-You need to install system libraries, as described in {ref}`plone:plone-prerequisites-label`, with the exception of GNU make.
+You need to install system libraries, as described in {ref}`plone:plone-prerequisites-label`.
 
-### tox
-
-[tox](https://tox.wiki/en/stable/index.html) automates and standardizes testing in Python.
-Install tox into your Python user space with the following command.
-
-```shell
-python -m pip install --user tox
-```
-
-### pre-commit
-
-`plone.api` uses [pre-commit](https://pre-commit.com/) to automate code quality checks before every commit.
-
-Install pre-commit either with your system package manager.
-Alternatively you can install pre-commit into your Python user.
-
-```shell
-python -m pip install --user pre-commit
-```
-
-Once installed, set up the git hook scripts to run on every commit.
-
-```shell
-pre-commit install
-```
 
 ## Create development environment
 
 After satisfying the prerequisites, you are ready to create your development environment.
 `plone.api` uses `tox` as a wrapper around `coredev.buildout` to simplify development, whereas Plone core uses `coredev.buildout` directly.
 
-Start by changing your working directory to your project folder, and download the latest `plone.api` source code.
+Start by changing your working directory to your project folder, and check out the latest `plone.api` source code.
 
 ```shell
 cd <your_project_folder>
 git clone https://github.com/plone/plone.api.git
 ```
 
-Next go into the newly created directory, and build your environment.
+Run `make help` to see the available `make` commands.
 
-```shell
-cd plone.api
-tox
+```console
+check                          Check code base according to Plone standards
+clean                          Clean environment
+help                           This help message
+livehtml                       Build docs and watch for changes
+test                           Run tests
 ```
 
-Go make some tea while `tox` runs all tasks listed by issuing the command `tox -l`.
-
-Open `tox.ini` in your code editor to see all configured commands and what they do.
-Some helpful `tox` commands are shown below.
+Test your code changes with the following command.
```

Comment:
Reading this flow, we jump from clone the repo, to `make help`, to running tests, whereas the section heading is "Create development environment". I expect a step that does just that. Of course some of the `make` commands do that, but you would only know that by reading `Makefile` and understanding its syntax.

```suggestion
The `make` commands `check`, `livehtml`, and `test` will create a development environment, if it does not already exist.

Test your code changes with the following command.
```

Reply:
better. even with this enhancement the isntructions are pretty short.